test/e2e/tests/test_tenant_model_inference.py

```
@@ -274,3 +279,149 @@ def test_tenant_isolation_cross_gateway_blocked(self, tenant_inference_cases):
+        """Wrong model name in body is rejected by IPP model-provider-resolver."""
+        case_a, _ = tenant_inference_cases
+        gateway_url = _get_tenant_gateway_url(case_a["gateway_name"])
+        api_key = _create_tenant_api_key(gateway_url, case_a)
+        _wait_reconcile()
+
+        r = self._post_chat(gateway_url, case_a["model_path"], api_key, {
+            "model": "nonexistent-model",
+            "messages": [{"role": "user", "content": "hello"}],
+        })
+        assert r.status_code != 200, (
+            f"Expected rejection for wrong model in body, got 200. "
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)